<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombatWindow</class>
 <widget class="QFrame" name="CombatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>701</width>
     <height>161</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="mainPage">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item row="1" column="1">
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="fightButton">
        <property name="text">
         <string>Fight</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="fightBuinventoryButtontton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Inventory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="switchButton">
        <property name="text">
         <string>Change Pokemon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="attackPage">
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item row="1" column="1">
       <widget class="QPushButton" name="attack4Button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Attack 4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="attack2Button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Attack 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="attack1Button">
        <property name="text">
         <string>Attack 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="attack3Button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Attack 3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="partyPage"/>
   <widget class="QWidget" name="inventoryPage">
    <widget class="QLabel" name="inventoryLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Inventory</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pokeballLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pokeball(s): 10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pokeballButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>54</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Use</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="dialoguePage">
    <widget class="QLabel" name="dialogueLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>661</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabelvr</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="fightView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="playerActiveLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:
Lvl: 1    HP 100/100</string>
   </property>
  </widget>
  <widget class="QLabel" name="opponentActiveLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:
Lvl: 1    HP 100/100</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
